scss/_patterns_notifications.scss

```diff
@@ -110,6 +116,5 @@ $response-icon-space: 2 * $sph-inner + map-get($icon-sizes, default);
 @mixin vf-notifications-information {
   .p-notification--information {
     @extend %vf-notification;
-    @include vf-highlight-bar($color-information);
```

Comment:
We should keep the blue highlight on the left here: 

```scss
@include vf-highlight-bar($color-information);
```

Also, information notification didn't have any icon previously, but for consistency it should also have information icon (same as default notification):

```scss
.p-notification__response {
    @include vf-icon-information;
}
```